api.py

```diff
@@ -1,11 +1,97 @@
-from flask import Flask
+import json
+import os
+import urllib
```

Comment:
Minor but maybe just import the actual methods rather than the whole lib?

Reply:
Added.